tools/debugstream/thread-info.py

```diff
@@ -1,217 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Does this need squashed into previous patch ?

Reply:
Actually it looks like I accidentally squashed some py-script-commit to an earlier patch... will fix.